.github/workflows/js/mage.yaml

```diff
@@ -0,0 +1,76 @@
+name: Lint, Test, Build and Publish to GitHub Package Registry
```

Comment:
if we can merge this into .github/workflows/mage that would be great. First see the comment in targets/jslib/main.go